brainscore_language/models/lm1b/__init__.py

```diff
@@ -0,0 +1,308 @@
+        return surprisal.item()
+
+    def predict_next_word(self, logits):
+        """
+        :param logits: the neural network's softmax output (seq_len, vocab_size)
+        :return: predicted next word
+        """
+
+        logits[:, 2] = 0 # mask <UNK> token for next word predictions
+        pred_id = np.argmax(logits, axis=1)
+        
+        # Note that this is currently only predicting the next *token* which might not always be entire words.
+        last_model_token_inference = pred_id[-1]
+        next_word = self.vocab.id_to_word(last_model_token_inference)
+        
+        # `next_word` often includes a space ` ` in front of the actual word. Since the task already tells us to output
+        # a word, we can strip the space.
+        next_word = next_word.strip()
+        return next_word
+    
+    def output_to_representations(self, layer_representations, stimuli_coords):
+        """Format layer representations as a neuroid assembly"""
+        
+        # Re-format layer_representations to contain region, recording type, and layer
+        layer_reps_mapping = {}
+        for layer_i, representations in enumerate(layer_representations):
+            recording_target, recording_type = self.neural_recordings[layer_i]
+            layer = self.region_layer_mapping[recording_target]
+            
+            key = (recording_target, recording_type, layer)
+            layer_reps_mapping[key] = representations
+        layer_representations = layer_reps_mapping
+
+        representation_values = np.concatenate(list(layer_representations.values()), axis=-1)  # concatenate along neuron axis
+        neuroid_coords = {
+            'layer': ('neuroid', np.concatenate([[layer] * values.shape[-1]
+                                                 for (recording_target, recording_type, layer), values
+                                                 in layer_representations.items()])),
+            'region': ('neuroid', np.concatenate([[recording_target] * values.shape[-1]
+                                                  for (recording_target, recording_type, layer), values
+                                                  in layer_representations.items()])),
+            'recording_type': ('neuroid', np.concatenate([[recording_type] * values.shape[-1]
+                                                          for (recording_target, recording_type, layer), values
+                                                          in layer_representations.items()])),
+            'neuron_number_in_layer': ('neuroid', np.concatenate(
+                [np.arange(values.shape[-1]) for values in layer_representations.values()])),
+        }
+        neuroid_coords['neuroid_id'] = 'neuroid', functools.reduce(defchararray.add, [
+            neuroid_coords['layer'][1], '--', neuroid_coords['neuron_number_in_layer'][1].astype(str)])
+        
+        representations = NeuroidAssembly(
+            representation_values, # 1024
```

Comment:
```suggestion
            representation_values,
```
not sure what the comment is about